lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -34,6 +34,12 @@ nv_overlay_evpn:
   get_value: '/^nv overlay evpn$/'
   set_value: "nv overlay evpn"
 
+pim:
+  kind: boolean
+  get_command: 'show running | section feature'
```

Comment:
Nit: the other get_command's are doing `'show running | i ^feature'`.

I was also thinking we could just use a top-level get_command for most of these. Let's consider changing that on the next fix to this file.